packages/features/portfolio/src/containers/portfolio-risk/portfolio-risk-mobile.tsx

```diff
@@ -0,0 +1,54 @@
+import { observer } from 'mobx-react-lite';
+import { Box, styled, useTheme } from '@mui/material';
+import { Body, HeaderToggle } from '@notional-finance/mui';
+import { FormattedMessage } from 'react-intl';
+import { useState } from 'react';
+import PortfolioTab from './components/portfolio-tab';
+import VaultTab from './components/vault-tab';
+
+const PortfolioRiskMobile = () => {
+  const theme = useTheme();
+  const [selectedTabIndex, setSelectedTabIndex] = useState(0);
+
+  return (
+    <Box>
+      <ToggleContainer>
+        <HeaderToggle
```

Comment:
Determine if the account has vault or portfolio holdings and only show the toggle if it has both. If it only has one or the other just show the relevant tab:

You can determine the relevant states using this code:

```
const account = useAccountDefinition(network);
const hasPortfolioRisk = !!account?.balances.find(
    (t) => t.isNegative() && !t.isVaultToken
  );
const hasVaultRisk = !!account?.balances.find(
    (t) => t.isVaultToken
  );
```